### PR TITLE
Add standing loss param for thermal storage

### PR DESCRIPTION
**Pull request by TomKae00**
## Changes proposed in this Pull Request

- Added standing losses as parameters for PTES, central TTES, decentral TTES 

## Checklist

- [x] Code changes are sufficiently documented; i.e. new functions contain docstrings and further explanations may be given in `doc`.
- [x] Data source for new technologies is clearly stated.
- [x] Newly introduced dependencies are added to `environment.yaml` (if applicable).
- [x] A note for the release notes `doc/release_notes.rst` of the upcoming release is included.
- [x] I consent to the release of this PR's code under the GPLv3 license.

### CODE REVIEW

File: scripts/compile_cost_assumptions.py [hunk 1]
```diff
@@ -2190,6 +2191,15 @@ def order_data(years: list, technology_dataframe: pd.DataFrame) -> pd.DataFrame:
             clean_df[tech_name] = pd.concat(
                 [clean_df[tech_name], bottom_storage_temp_ptes]
             )
+            energy_loss = df.loc[
+                df.index.str.contains("Energy losses during storage")
+            ].copy()
+            energy_loss["parameter"] = "Energy losses during storage"
+            energy_loss[years] = (
+                energy_loss[years] / 78 * 100 / 24
```
Comment (cpschau) [inline scripts/compile_cost_assumptions.py:2199]: I think the formula must also take into account the minimum temperature temperature of the storage at SOC=0, so the resulting percentage value of the standing loss has the right reference.

**Review by cpschau — CHANGES_REQUESTED** [review]

File: scripts/compile_cost_assumptions.py (hunk 1 above)
Comment (TomKae00) [reply scripts/compile_cost_assumptions.py]: Alright, I've now set the average PTES storage temperature relative to the bottom storage temperature, which represents the SOC=0.

Comment (cpschau) [reply scripts/compile_cost_assumptions.py]: Great! There still seems to be some problem with the calculation or the compilation, as there is no value written into the cost files for PTES.

Comment (TomKae00) [reply scripts/compile_cost_assumptions.py]: Thanks for pointing that out! I’ve fixed the issue, PTES values are now correctly written into the csv cost files.

File: outputs/US/costs_2020.csv [hunk 2]
```diff
@@ -3721,13 +3721,15 @@ central solid biomass CHP powerboost CC,lifetime,25.0,years,"Danish Energy Agenc
...
 central water pit storage,energy to power ratio,150.0,h,"Danish Energy Agency, inputs/technology_data_catalogue_for_energy_storage.xlsx",140 PTES seasonal:  Ratio between energy storage and input capacity,2020.0,,
 central water pit storage,investment,1.0622,EUR/kWh Capacity,"Danish Energy Agency, inputs/technology_data_catalogue_for_energy_storage.xlsx",140 PTES seasonal:  Specific investment,2020.0,,
 central water pit storage,lifetime,20.0,years,"Danish Energy Agency, inputs/technology_data_catalogue_for_energy_storage.xlsx",140 PTES seasonal:  Technical lifetime,2020.0,,
 central water tank charger,efficiency,1.0,per unit,"Danish Energy Agency, inputs/technology_data_catalogue_for_energy_storage.xlsx",:  Charger efficiency,2020.0,,
 central water tank discharger,efficiency,1.0,per unit,"Danish Energy Agency, inputs/technology_data_catalogue_for_energy_storage.xlsx",:  Discharger efficiency,2020.0,,
+central water tank storage,Energy losses during storage,0.0077,per unit,"Danish Energy Agency, inputs/technology_data_catalogue_for_energy_storage.xlsx",141 Large hot water tank:  Energy losses during storage,2020.0,,
```
Comment (cpschau) [inline outputs/US/costs_2020.csv:3732]: Stumbled upon this value in the last second. If I divide the daily percentual losses of large hot water tanks in the DEA catalogue by 24 I yield 0.2/24 = 0.0083. The computation in `compile_cost_assumptions` looks alright though. Any ideas why the values deviate?

**Review by cpschau — CHANGES_REQUESTED** [review]
LGTM and can be merged!

**Review by cpschau — APPROVED** [review]

**Comment by fneum** [comment]
Perhaps one detail you could add: What is the unit? Not quite sure what the "per unit" refers to here.
